app/src/main/java/org/piramalswasthya/sakhi/helpers/Konstants.kt

```diff
@@ -26,7 +26,7 @@ object Konstants {
     const val minAgeForNcd: Int = 30
     const val minAgeForReproductiveAge: Int = 15
     const val maxAgeForReproductiveAge: Int = 49
-    const val maxAgeForInfant: Int = 42
+    const val maxAgeForInfant: Int = 61
```

Comment:
These number validations don't make any real world sense. 
cc: @madhavaramu83

Reply:
This field is being used for some validations.

Reply:
@drtechie , This age value is in days.

Reply:
neonate - upto 28 days.
infant is more > 28 days to 1 year of age. 

so where did this 61 days come from? 🤔

Reply:
@drtechie , We are using this for validation. We are moving the record to HBNC after 61 days.

Reply:
Alright, okay. If the validation is used in right context, it's fine.

Reply:
yes, @sushant-bizbrolly it should be "neonate" not "infant", 
but here the logic is keeping the records up 61 days in the HBNC section for the User to do data entry.